lib/components/Forbidden403/Forbidden403.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+import { Forbidden403 } from './Forbidden403'
+import { ButtonGroup } from '../ButtonGroup/ButtonGroup'
+
+const meta: Meta<typeof Forbidden403> = {
+  title: 'Components/Forbidden403',
```

Comment:
Lets add an extra path here to group all error components, so the title should be `Components/ErrorMessages/Forbidden403`

Reply:
done